apps/src/javalab/JavalabView.jsx

```diff
@@ -1,17 +1,23 @@
+import $ from 'jquery';
 import React from 'react';
 import PropTypes from 'prop-types';
-import i18n from '@cdo/locale';
```

Comment:
This file is starting to seem like it does too much. Is there a logical way to split it into different functionalities? One possibility: It contains logic for button clicks and for resizing. Maybe these two can be separated?

Reply:
That would be great, but I think best pushed to a future PR that focuses on refactoring.

Reply:
Makes sense. Mind adding a task?

Reply:
Done.